output/summary.go

```diff
@@ -38,6 +41,27 @@ type Summary struct {
 }
 
 func (s *Summary) Start(plan *plan.Plan) error {
+	config := plan.Config
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 8, 0, ' ', tabwriter.AlignRight)
+	defer w.Flush()
+
+	fmt.Fprintf(w, "Configuration:\n")
```

Comment:
I would move the config output into it's own reporter, users choosing the summary output probably don't want to see this, and if they should be able to just:

```
REPORTER=config,summary
```